apps/summerfi-api/lib/sdk-router-function/src/server.ts

```diff
@@ -0,0 +1,48 @@
+import { initTRPC } from '@trpc/server'
+import { z } from 'zod'
+import {
+  isPositionId,
+  isProtocolEnum,
+  Position,
+  Protocol,
+  type PositionId,
+  type ProtocolEnum,
+} from './sdk-mocks'
+
+/**
+ * Server
+ */
+
+const t = initTRPC.create()
+export const router = t.router
+export const publicProcedure = t.procedure
+export const createCallerFactory = t.createCallerFactory
+
+export const appRouter = router({
```

Comment:
Not necessary now, but moving forward we should think of a way of organizing this functions. Perhaps in different files and linked them here, for example

Reply:
I updated structure a bit to be better organized let me WDYT :)